view/frontend/web/js/view/payment/method-renderer/adyen-cc-method.js

```diff
@@ -384,6 +401,9 @@ define(
                     // render component
                     self.renderThreeDS2Component(response.type, response.token, orderId);
                 } else {
+                    if(response.type==='RedirectShopper' ){
+                        self.threedsfallback( response.action);
+                    }
                     window.location.replace(url.build(
                         window.checkoutConfig.payment[quote.paymentMethod().method].redirectUrl
                     ));
```

Comment:
The JS doesn't stop when calling the threedsfallback() so I would make sure that the window.location.replace is not running anymore if the response.type is RedirectShopper
```suggestion
                    } else {
                        window.location.replace(url.build(
                            window.checkoutConfig.payment[quote.paymentMethod().method].redirectUrl
                        ));
                    }
```